provider/terraform/custom_code.rb

```diff
@@ -146,11 +146,13 @@ def oics_link
       end
 
       def substitute_test_paths(config)
+        config = config.gsub('../static/img/header-logo.png', 'test-fixtures/header-logo.png')
```

Comment:
Why does the png need to be checked into source control?

Reply:
We need ~something~ to send up to our bucket, and I prefer sending "real" files up to just generating the file at test time. We already have a few static files for ssl certs and cloud functions, so I figure this image can be too.

Reply:
As long as we keep the file sizes small and don't change them much it shouldn't be a big deal. Big or frequently changing binary files make git sad.

Reply:
oh yup it is a 16x16 grey image very much on purpose haha. imo tossing 1-2 small images into vcs is fine, if it were any more/any larger it would 100% be time for a more future-proof solution